<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QTabWidget" name="input_filter_scientists">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_computer">
       <attribute name="title">
        <string>Computers</string>
       </attribute>
       <widget class="QLineEdit" name="input_computer_was_built">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="list_computers">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>81</y>
          <width>551</width>
          <height>79</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>541</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_computer_name_2">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_error_computer_name_2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QComboBox" name="dropdown_order_by_computers">
        <property name="geometry">
         <rect>
          <x>3</x>
          <y>49</y>
          <width>561</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_computer_name">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>480</y>
          <width>561</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_computer_year_built">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>450</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_filter_computers">
        <property name="geometry">
         <rect>
          <x>6</x>
          <y>20</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Type search term here</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_remove_computer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>561</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove selected computer</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="input_computer_type">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10"/>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_computer_type_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>41</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_computer_was_built_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>547</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>WasBuilt:</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>370</y>
          <width>549</width>
          <height>19</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>420</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_computer_year_built_2">
           <property name="text">
            <string>YearBuilt:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_error_computer_year_built_2">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>541</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_add_computer_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Add a new computer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>420</y>
          <width>551</width>
          <height>21</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9"/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_scientist">
       <attribute name="title">
        <string>Scientists</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>561</width>
          <height>21</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Type search term here</string>
        </property>
       </widget>
       <widget class="QComboBox" name="dropdown_order_by_scientists">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>561</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="table_scientists">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>561</width>
          <height>231</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="button_remove_scientist">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>310</y>
          <width>561</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove selected scientist</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>360</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>Add scientist</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>FirstName</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
